recipes/arrow-c-glib/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [[ ${OSTYPE} == 'darwin'* ]]; then
+  export XML_CATALOG_FILES="${PREFIX}/etc/xml/catalog"
+fi
```

Comment:
I think that we don't need this because GTK-Doc isn't enabled by `-Dgtk_doc=true`.